src/gcp_scanner/crawler/misc_crawler.py

```diff
@@ -1,41 +1,25 @@
-# Copyright 2022 Google LLC
+#  Copyright 2023 Google LLC
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Do we really need this extra space here and below?

Reply:
Updated.